EncodingStreamingHighAvailability/media-services-high-availability-shared/Models/JobOutputModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace media_services_high_availability_shared.Models
+{
+    using Microsoft.Azure.EventGrid.Models;
+    using Newtonsoft.Json;
+
+    public class JobOutputModel : MediaJobOutput
+    {
+        public JobOutputModel()
+        {
+            this.OutputAssetName = string.Empty;
+        }
+
+        [JsonProperty(PropertyName = "assetName")]
```

Comment:
Why can't you use https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Microsoft.Azure.EventGrid/src/Generated/Models/MediaJobOutputAsset.cs here?  Meaning it should already be included in the message from EventGrid.  Are you not seeing that?